new_dawn_server/users/api/resources.py

```diff
@@ -326,6 +329,20 @@ def get_and_save_answer_question(bundle, user, profile):
                 )
                 answer_question.save()
 
+    @staticmethod
+    def get_and_save_images(bundle, user, profile):
+        images_bundle = bundle.data.get("images")
+        if images_bundle:
+            for single_image_bundle in images_bundle:
+                image_data = Image(
+                    caption=single_image_bundle["caption"],
+                    media=single_image_bundle["media"],
```

Comment:
Image upload API should be separated from main registration flow since they are really slow. iOS should store user's own images locally and use them at first after registration and send an async call to the standalone image upload API to backend. I've implemented an image upload API already in medias/api/ to handle image upload request (which is a multi-part request and you can find more about how to construct that using Alamofire online such as https://stackoverflow.com/questions/42642235/upload-a-photo-to-django-in-swift-3-using-alamofire-4).

Reply:
Sure this sounds good to me.